src/lib/AnnotationModule.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    bdlBoxBlue,
+    bdlGreenLight,
+    bdlWatermelonRed,
+    bdlYellorange,
+    bdlYellow,
+    bdlGrimace,
+} from 'box-ui-elements/es/styles/variables';
+import Cache from './Cache';
+
+export const ANNOTATION_COLOR_KEY = 'annotation-color';
+
+export const AnnotationColor = {
+    BOX_BLUE: bdlBoxBlue,
+    GREEN_LIGHT: bdlGreenLight,
+    WATERMELON_RED: bdlWatermelonRed,
+    YELLORANGE: bdlYellorange,
+    YELLOW: bdlYellow,
+    GRIMACE: bdlGrimace,
+};
+
+export default class AnnotationModule {
+    private cache: Cache;
+
+    constructor({ cache }: { cache: Cache }) {
+        this.cache = cache;
+
+        if (!this.getColor()) {
+            this.setColor(AnnotationColor.BOX_BLUE);
+        }
+    }
+
+    public getColor = (): string => this.cache.get(ANNOTATION_COLOR_KEY) as string;
```

Comment:
Do we need to set the default value in the cache in the constructor or can we return it here with `|| bdlBoxBlue`?

Reply:
Nope, we can use the OR with `bdlBoxBlue`.